cdap-ui/app/cdap/components/PipelineScheduler/Store/ActionCreator.js

```diff
@@ -31,6 +31,9 @@ import { GLOBALS, CLOUD } from 'services/global-constants';
 import { Observable } from 'rxjs/Observable';
 
 function setStateFromCron(cron = PipelineSchedulerStore.getState().cron) {
+  if (!cron) {
+    return;
```

Comment:
For follow up: Can we add a unit test to cover this case?

Reply:
Will do. Since we are not closing the JIRA we can add when during the investigation.